src/PageView/Elements/Elements.js

```diff
@@ -24,7 +24,7 @@ export default function Elements() {
                 <h1 className="separator-right">Left Separator</h1>
             </div>
             <div className="grid">
-                <div>
+                {/* <div>
```

Comment:
Is the reason you commenting this out the warnings you get in the console?

Reply:
yes that is why

Reply:
Don't think is relevant with what you are trying to address here and they are only warnings.

If you would like to address this here I would suggest you replace '#' with '/'